packages/common/test/hardforks.spec.ts

```diff
@@ -305,10 +305,10 @@ describe('[Common]: Hardfork logic', () => {
         terminalTotalDifficulty: 0,
         shanghaiTime: 0,
         cancunTime: 0,
-      },
-      difficulty: '100',
+      } as GethGenesisConfig,
+      difficulty: '0x100',
```

Comment:
Nit: 100 != 0x100. This change might not be part of tests, so might be fine

Reply:
Yeah I just changed those to whatever value because they were actually invalid and just unused placeholders.